src/include/daos_srv/daos_engine.h

```diff
@@ -234,6 +234,17 @@ void sched_req_sleep(struct sched_request *req, uint32_t msec);
  */
 void sched_req_wakeup(struct sched_request *req);
 
+/* clang-format off */
+/**
+ * Abort a sched request attached ULT (without waiting).
+ *
+ * \param[in] req	Sched request.
+ *
+ * \retval		N/A
+ */
+void sched_req_abort(struct sched_request *req);
```

Comment:
As I commented above, I think we'd try to avoid introducing new API unless it's really necessary.